### ISSUE TITLE
Player.isTalking property doesn't work

### ISSUE THREAD

**Issue by targaryen94** [altmp/altv#417]
## Expected Behavior
if i speaking return true value.

## Current Behavior
always return false value

## Steps to Reproduce
1. create client side file
2. in write setInterval(() => {  alt.log(alt.Player.local.isTalking)}, 5000)

## Detailed Description

I found this pages:
https://wiki.altv.mp/ClientJS:Player
https://wiki.altv.mp/NetworkIsPlayerTalking

I using player.isTalking and networkisplayertalking but always return false value.
How to use networkisplayertalking? What is arguments? player.Script/player.id or what?

Other players hear me and they hear me ... so the voice chat is working,  functions not working.

**Comment by derRAV3N** [comment]
Hey, I think you shouldn't be using [NetworkIsPlayerTalking](https://natives.altv.mp/#/0x031E11F3D447647E) for this usage, as alt-voice is decoupled from the game itself, at least I think so.

I can't tell why [isTalking](https://altmp.github.io/altv-typings/classes/_alt_client_.player.html#istalking) is not working, but just FYI, you shouldn't use the wiki for natives and alt functions anymore, as there are these two pages now:
https://altmp.github.io/altv-typings/
https://natives.altv.mp/#/

**Comment by targaryen94** [comment]
Okay thank you for help.

**Comment by lameule123** [comment]
This issue is present in alpha-branch.

**Comment by Lhoerion** [comment]
Added/fixed/changed in build #1004

**Comment by datWeazel** [comment]
Seems like the error is back. (Client 1031, Branch 'rc')
```
alt.everyTick(() => {
        alt.log("alt.Player.local.isTalking: " + alt.Player.local.isTalking);
});
``` 
Using these settings (https://i.imgur.com/GjZiDDY.png) the output is always `alt.Player.local.isTalking: false` Neither if I push and hold "N" or if I just talk. As you can see my mic is recognized by altV. Sadly I couldn't test if others can hear me or not.
